src/notifier.js

```diff
@@ -160,7 +151,7 @@ NotifierPrototype._getLogArgs = function(args) {
     } else if (argT === 'date') {
       extraArgs.push(arg);
     } else if (argT === 'error' ||
-               arg.stack ||
+               arg instanceof Error ||
```

Comment:
This looks like it will break things if any users were passing in an object with a `stack` property. I'm not sure if anyone is doing this but it seems possible. Any reason not to still support it? If so, we should bump the version number (minor version should be fine, since it's not part of the public api).